Microsoft.Toolkit.Uwp.SampleApp/Controls/SampleAppMarkdownRenderer.cs

```diff
@@ -1,518 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Microsoft.Toolkit.Parsers.Markdown;
-using Microsoft.Toolkit.Parsers.Markdown.Blocks;
-using Microsoft.Toolkit.Parsers.Markdown.Inlines;
-using Microsoft.Toolkit.Parsers.Markdown.Render;
-using Microsoft.Toolkit.Uwp.Helpers;
-using Microsoft.Toolkit.Uwp.UI.Controls.Markdown.Render;
-using Windows.ApplicationModel.DataTransfer;
-using Windows.UI;
-using Windows.UI.Text;
-using Windows.UI.Xaml;
-using Windows.UI.Xaml.Controls;
-using Windows.UI.Xaml.Media;
-
-namespace Microsoft.Toolkit.Uwp.SampleApp.Controls
-{
-    /// <summary>
-    /// A Rendering Superclass for the Markdown Renderer, allowing custom styling of Elements in Markdown.
-    /// </summary>
-    internal class SampleAppMarkdownRenderer : MarkdownRenderer
```

Comment:
@RosarioPulella this is what helps display the docs within the Sample App, so I don't think we can remove it? Is it because the `MarkdownRenderer` would be internalized now? That could be a problem... I think we'd still want that class exposed?

Or did this just move somewhere else?